.github/workflows/ci.yml

```diff
@@ -1,27 +1,27 @@
-name: Run Gleam tests and verify README examples
+name: Test, ensure formatted, and verify README examples
 
 on: [push]
 
 jobs:
-  test_and_verify_readme_examples:
+  ci:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2.0.0
 
-      - uses: gleam-lang/setup-erlang@v1.0.0
+      - uses: gleam-lang/setup-erlang@v1.1.0
         with:
           otp-version: 22.1
```

Comment:
I tried to update this to `23.0`, but the Action was failing to find the release on Erlang Solutions. We'll try again later.